src/azul/chalice.py

```diff
@@ -189,15 +189,33 @@ def _api_gateway_context_middleware(self, event, get_response):
         finally:
             config.lambda_is_handling_api_gateway_request = False
 
+    hsts_max_age = 60 * 60 * 24 * 365 * 2
+
+    security_headers = {
+        # FIXME: Use join_words()
+        #        https://github.com/DataBiosphere/azul-private/issues/6
+        'Strict-Transport-Security': f'max-age={hsts_max_age};'
+                                     ' includeSubDomains;'
+                                     ' preload',
+        'X-Content-Type-Options': 'nosniff',
+        'X-Frame-Options': 'DENY',
+    }
+
+    default_response_headers = {
```

Comment:
A comment should explain why they are segregated.

Reply:
The name of the attribute contradicts the comment. Given the name of the variable, one would expect a "default" to be included in other definitions. You include the other definition in the "default".